Camunda.Api.Client/UserTask/CompleteTaskBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Camunda.Api.Client.UserTask
+{
+    public abstract class CompleteTaskBase
+    {
+        /// <summary>
+        /// Variables will return after completing the task
+        /// </summary>
+        public abstract bool WithVariablesInReturn { get; }
```

Comment:
protected or internal?

Reply:
this musst be public because only public properties are serialized to json